cmd/otelcontribcol/go.mod

```diff
@@ -571,8 +572,8 @@ require (
 	go.opentelemetry.io/collector/confmap v0.70.1-0.20230130215412-26bd7b2bf678 // indirect
 	go.opentelemetry.io/collector/consumer v0.70.1-0.20230130215412-26bd7b2bf678 // indirect
 	go.opentelemetry.io/collector/featuregate v0.70.1-0.20230130215412-26bd7b2bf678 // indirect
-	go.opentelemetry.io/collector/pdata v1.0.0-rc4.0.20230130215412-26bd7b2bf678 // indirect
-	go.opentelemetry.io/collector/semconv v0.70.1-0.20230130215412-26bd7b2bf678 // indirect
+	go.opentelemetry.io/collector/pdata v1.0.0-rc5 // indirect
+	go.opentelemetry.io/collector/semconv v0.71.0 // indirect
```

Comment:
This is already updated on main by fac9f8ba05b3af51e4e11e2cb639db3a4b7202d0, you need to rebase

Reply:
Oh my, I was way far behind. Sorry about that.